src/d3a/setup/nested_grid_test.py

```diff
@@ -0,0 +1,64 @@
+"""
```

Comment:
This setup file is not used in the code. Do you have another reason for adding it?